frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/capacity-and-nodes.tsx

```diff
@@ -23,18 +23,16 @@ import { arbiterText } from '../../../constants';
 import { getZone, isArbiterSC } from '../../../utils/install';
 import { AdvancedSubscription } from '../subscription-icon';
 
-export const SelectNodesText: React.FC<SelectNodesTextProps> = React.memo(({ text, replica }) => {
+export const SelectNodesText: React.FC<SelectNodesTextProps> = React.memo(({ text }) => {
```

Comment:
`replica` is no longer required.
Remove from types as well:
```
type SelectNodesTextProps = { text: string; replica: number };
```

Reply:
@afreen23 ohh! yeah!